lifecycle-operator/apis/lifecycle/v1alpha3/common/common.go

```diff
@@ -23,15 +23,17 @@ const K8sRecommendedAppAnnotations = "app.kubernetes.io/part-of"
 const K8sRecommendedManagedByAnnotations = "app.kubernetes.io/managed-by"
 const PreDeploymentEvaluationAnnotation = "keptn.sh/pre-deployment-evaluations"
 const PostDeploymentEvaluationAnnotation = "keptn.sh/post-deployment-evaluations"
+const SchedulingGateRemoved = "keptn.sh/scheduling-gate-removed"
 const TaskNameAnnotation = "keptn.sh/task-name"
 const NamespaceEnabledAnnotation = "keptn.sh/lifecycle-toolkit"
 const CreateAppTaskSpanName = "create_%s_app_task"
 const CreateWorkloadTaskSpanName = "create_%s_deployment_task"
 const CreateAppEvalSpanName = "create_%s_app_evaluation"
 const CreateWorkloadEvalSpanName = "create_%s_deployment_evaluation"
 const AppTypeAnnotation = "keptn.sh/app-type"
+const KeptnGate = "gated-keptn"
```

Comment:
final name change, I promised 😇 
This way, if someone inspects a pod and sees the injected gates, the name describes better the purpose of them 
```suggestion
const KeptnGate = "keptn-prechecks-gate"
```